Aptoide/aptoide-client-v8
# [feature/ASV-1866] - Appc View Detail

**What does this PR do?**

   This PR implements the new APPC info view. Check Zeplin for reference.

**Database changed?**

   No

**How should this be manually tested?**

  Go to the new appc view (you can through the Earn AppCoins bundle) and check that graphically everything is ok and that video works correctly. When there's no internet, there's a error for it but that view will be redone later.

**What are the relevant tickets?**

  [ASV-1866](https://aptoide.atlassian.net/browse/ASV-1866)


**Code Review Checklist**
